android/src/main/java/com/reactnativecommunity/webview/RNCWebView.java

```diff
@@ -40,6 +40,7 @@
 import com.reactnativecommunity.webview.events.TopCustomMenuSelectionEvent;
 import com.reactnativecommunity.webview.events.TopLoadingProgressEvent;
 import com.reactnativecommunity.webview.events.TopMessageEvent;
+import com.reactnativecommunity.webview.events.TopOpenWindowEvent;
```

Comment:
Can this import be removed now?

Reply:
Fixed in: https://github.com/react-native-webview/react-native-webview/compare/5a835821371ba93f5a5ac120c88097fc3c918ff4..610aee3309dfd69739c28a5bb4579b91015d0394